packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -627,14 +627,27 @@ const convertPathToHoppReqs = (
           ? openAPIUrl + openAPIPath.slice(1)
           : openAPIUrl + openAPIPath
 
+      let defaultRequestName = "Untitled Request"
+
+      if (info.tags?.length) {
+        const pathStartIndex =
+          openAPIPath.indexOf(info.tags[0]) + info.tags[0].length
+
+        if (pathStartIndex > 0) {
+          defaultRequestName =
```

Comment:
when naming the request, here it seems we're considering the first value after the tag only, eg: the current logic will name `/categories/<<categoryID>/update` as `<<categoryID>>`, but i would prefer the request to be named, whatever the remaining path is after the tag. in this case `<<categoryID>>/update`